### PR TITLE
Static Todo App

### PR DESCRIPTION
**Pull request by DiamondNath07**
### Essence of this PR
This PR shows the various list of items added to the todo app
It also makes use of webpack for hosting the application
It also sends feedback of list added to the list to aid user's experience

### This app was built with the following technologies
- index.html
- styles.css
- index.js
- webpack

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌
# Hello @DiamondNath07  👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where you have not attached the required linters. Kindly attach the required linters and submit for a review again. 👍🏻 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @DiamondNath07, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, without active linters
Please follow the [microverse linter guide](https://github.com/microverseinc/linters-config/tree/master/javascript) and add try everything possible to make your linter active before submitting again for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @DiamondNath07   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] The purpose of this milestone was to create a structure for the todos. Please create an array and insert the static task (must be an object) and iterate through the created array, on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

   structure example:
                          ` const tasks = [
                                                      {
                                                           description: 'An example description in the string',
                                                           completed: false,
                                                            index: 10,
                                                       },
                                                 ];`
             

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/todoapp.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  addActivity() {
+    const addtext = document.querySelector('.add-text');
+    const addList = {
+      id: Date.now(),
+      description: addtext.value,
+      index: this.todo.length + 1,
+      completed: false,
+    };
```
Comment (Olamarx) [inline src/modules/todoapp.js:20]: [OPTIONAL] You don't need to add an id, the index can serve as the id.

File: src/modules/todoapp.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+export default class TodoList {
+  constructor() {
+    this.todo = [];
```
Comment (Olamarx) [inline src/modules/todoapp.js:3]: - [ ] Kindly populate this array with some data to be rendered on the UI.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @DiamondNath07, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters passed ✅ 
- Git and GitHub flow 👍🏼 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — COMMENTED** [review]
Hello @DiamondNath07👋,

_This is an additional review as a result of the second opinion request._

Firstly, I would like you to note that sending one SO request is enough. It is not good to spam the system with several SO requests.

You have a concern in one of your SO requests that you don't understand the suggestion from @Olamarx. Second Opinion cannot help with this. You need to reach out to the Code Reviewer to get clarifications. Always remember this.

The problem with your project is that it's ahead of this milestone and that is why you are having problems with the code reviews you are receiving. You have gone ahead and created the functionality for adding a task. When you finally reach the milestone concerned with the functionality of adding a task, what will expect the code reviewer to review? This part of your code will be missing in the FIles Changed part of the PR and when issues arise, you'll be asking for another Second Opinion. I don't believe this is how you'd like things to be.

Please stick to the project requirements to avoid these small misunderstandings. Take a look at the screenshot of the project requirements below, read them carefully, and implement them.

image

At this point, just like @Olamarx suggested, please add some example tasks. When your app is loaded, these example tasks should be loaded on the page.

Please work on the concerns above, then ask for a code review from your dashboard.

Keep rocking 🚀!

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @DiamondNath07 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Linters passed ✅
 - Git and GitHub flow 👍🏼
 - Descriptive commit messages ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**